sky/engine/platform/BUILD.gn

```diff
@@ -97,7 +97,6 @@ source_set("platform") {
     "fonts/harfbuzz/HarfBuzzFaceSkia.cpp",
     "fonts/harfbuzz/HarfBuzzShaper.cpp",
     "fonts/harfbuzz/HarfBuzzShaper.h",
-    "fonts/linux/FontPlatformDataLinux.cpp",
```

Comment:
Btw not sure how this worked before as the combo of this + FontPlatformDataApple generates duplicate symbols on Mac.